data/dataexchange/src/main/java/tech/pegasys/teku/data/SlashingProtectionImporter.java

```diff
@@ -48,7 +48,7 @@
 
   public SlashingProtectionImporter(final SubCommandLogger log, final String path) {
     this.log = log;
-    syncDataAccessor = SyncDataAccessor.create(Paths.get(path));
+    syncDataAccessor = SyncDataAccessor.create(Paths.get(path), false);
```

Comment:
Path here is a directory right? Otherwise we should call getParent here too.

Reply:
I can double check that, but basically its ignored now anyway